test/execflags/bradc/callStackSize.comm-none.good.in

```diff
@@ -1 +1,2 @@
+callStackSize.chpl:3: warning: 'locale.callStackSize' is deprecated.
```

Comment:
I think it'd be reasonable to just remove this test rather than updating its output, since it looks to be testing the feature that is deprecated